### DISCUSSION TITLE
Login warning (Main-account login via "action=login") with logging configured

### DISCUSSION THREAD

**Discussion by ross-spencer** [LeMyst/WikibaseIntegrator#394] [Q&A] [answered]
Raising awareness of this, and wondering if it as immediate risk at all? We're seeing this when debugging is enabled in a test app. MediaWiki version: `1.36.3`.

```
WARNING:root:* login: Main-account login via "action=login" is deprecated and may stop working without warning. To continue login with "action=login", see [[Special:BotPasswords]]. To safely continue using main-account login, see "action=clientlogin".
```

**Comment by LeMyst** [comment, marked as answer]
Hello @ross-spencer 
It's a message generated by the Mediawiki API.
AFAIK it's not at immediate risk, but you should ask the wikitech list for more informations : https://lists.wikimedia.org/hyperkitty/list/wikitech-l@lists.wikimedia.org/

It's really easy to setup a BotPasswords, you just need to got to Special:BotPasswords, give a name for your bot, set some specials rights if needed, and the interface give you the login/password to use.
